### DISCUSSION TITLE
Dont re-release releases

### DISCUSSION THREAD

**Discussion by myoung34** [community/community#12225] [Actions]
2.888.0 came out roughly 4pm CST on 2/25/22

Today it re-released again with new SHA's https://github.com/actions/runner/releases/tag/v2.288.0

This messes with upstream automation since the GH runners show an update then the SHA's change for the code itself

It also means that third party releasers like myself that package it as the same version get API errors trying to override packages (which are immutable)

**Comment by ethomson** [comment]
Thanks @myoung34 - many apologies. This was an error and it won't happen again.

**Comment by myoung34** [comment]
@ethomson  This is still happening in a way. This one wasnt a re-release but the newest release is actually semantically a downgrade listed as `prerelease`

My automation picked up a release [here](https://api.github.com/repos/actions/runner/releases/64784402)

```
  "name": "v2.289.3",
  "draft": false,
  "prerelease": true,
  "created_at": "2022-04-19T20:30:36Z",
  "published_at": "2022-04-19T20:36:40Z",
```

Despite 2.289.3 being a downgrade: https://github.com/actions/runner/releases

2.289.3 shows `prerelease` over 2.290.1 from 6 days ago which is listed as `latest`

**Comment by myoung34** [comment]
@ethomson hey its your boy again....

Latest is 2.301.1 but

```
[10:08 AM] A new github/actions release v2.289.5 is out:
https://github.com/actions/runner/releases/tag/v2.289.5

[10:10 AM] A new github/actions release v2.299.2 is out:
https://github.com/actions/runner/releases/tag/v2.299.2

[10:10 AM] A new github/actions release v2.296.3 is out:
https://github.com/actions/runner/releases/tag/v2.296.3

[10:10 AM] A new github/actions release v2.293.2 is out:
https://github.com/actions/runner/releases/tag/v2.293.2
```

Looks like something just kicked off a bunch of retags from old versions

**Comment by joeyparrish** [comment]
This is breaking my Actions runners!  @ethomson, can this be fixed please?  Is there anything we can do to contribute to the fix?  Happy to take pointers and send PRs.
